app/assets/stylesheets/active_admin/components/_status_tags.scss

```diff
@@ -1,16 +1,15 @@
 .status_tag {
-  background: darken($secondary-color, 15%);
-  color: #fff;
-  text-transform: uppercase;
```

Comment:
Thanks! I disagree. I find them less readable in all uppercase. I tend to think acronyms when I see things in all uppercase like that. I switched as I found it much less thought process to determine what the content is, I don't feel like I'm being shouted at and less confusing next to content that is in normal case. Also comparing with other systems and frameworks, labels like these are written in that way. Nothing stopping anyone from changing that of course but I felt the better default was standard case.